torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -132,6 +132,7 @@ def aten_getitem(self, i):
     "aten::softmax": ops.special.aten_special_softmax,
     "aten::split": ops.core.aten_split,
     "aten::sqrt": ops.core.aten_sqrt,
+    "aten::stack": ops.core.aten_stack,
```

Comment:
@justinchuby iirc this op have been in torchlib for a while. Curious what is the updating routine for bringing in new supported ops in torchlib?

Reply:
I don’t think we have a routine for now. When we do dispatch it should become a discovery process

Reply:
For now they are just added as needed

Reply:
We could adopt fx exporter into `test_op_consistency`.